addons/account/models/partner.py

```diff
@@ -1023,3 +1023,29 @@ def _compute_partner_company_registry_placeholder(self):
         for partner in self:
             country_code = partner.country_id.code or ''
             partner.partner_company_registry_placeholder = _ref_company_registry.get(country_code.lower(), '')
+
+    def _get_stat_info(self):
+        data_list = super()._get_stat_info()
+        if not self.env.user.has_group('account.group_account_invoice'):
+            return data_list
+        account_stat = self._get_account_stat_count()
+        for partner in account_stat:
+            stat_info = {'iconClass': 'fa-pencil-square-o', 'value': account_stat.get(partner), 'label': _('Invoices/Bills/Mandates')}
+            data_list[partner.id].append(stat_info)
+        return data_list
+
+    def _get_account_stat_count(self):
```

Comment:
Maybe ``_compute_account_move_count``, like if it was a real compute like _compute_sale_order_count or other. Even if we have no field, well voila. We could even add the field itself :D .